include/zephyr/bluetooth/audio/bap.h

```diff
@@ -1665,6 +1685,15 @@ int bt_bap_broadcast_sink_stop(struct bt_bap_broadcast_sink *sink);
  */
 int bt_bap_broadcast_sink_delete(struct bt_bap_broadcast_sink *sink);
 
+/**
+ * @brief Find a broadcast sink by its Periodic Advertising sync
+ *
+ * @param sync Pointer to an advertising sync
+ *
+ * @return Pointer to the broadcast sink found
+ */
+struct bt_bap_broadcast_sink *broadcast_sink_get_by_pa(struct bt_le_per_adv_sync *sync);
```

Comment:
I had initially thought that this should just be an internal function, but I actually think it works well as a public function. Good idea :)

Reply:
@Thalley I think a public wrapper function here called `bt_bap_broadcast_sink_get_by_per_adv_sync` is better, which calls the static function `broadcast_sink_get_by_pa ` ?

Reply:
> @Thalley I think a public wrapper function here called `bt_bap_broadcast_sink_get_by_per_adv_sync` , which calls the original static function `broadcast_sink_get_by_pa ` is the best option ?

That could also work, but it would also be just fine to rename the original function to `bt_bap_broadcast_sink_get_by_per_adv_sync` :)